### DISCUSSION TITLE
Disable History reporting without disabling History

### DISCUSSION THREAD

**Discussion by UA-Archiver** [ReVanced/revanced-discussions#208] [Question]
Alright, so the title may sound a bit weird but let me explain
YouTube and YouTube Music are under one umbrella.
So, if you modify a history setting in one, it will sync over to the other automatically.
Even in MyActivity on Google, they both are shown together with no easy was to filter one or the other

This makes it impossible to have History for one enabled and disabled for the other (say enabled for normal YT but disable in YTM) and you have to manually go and delete the items from history with a browser

So, the ability to suspend reporting of History will be useful

**Comment by oSumAtrIX** [comment]
Suspending is already possible by pausing the history.

**Reply by UA-Archiver** [reply]
I know that but its synced cross both YT and YTM and to all devices

What I am suggesting for the patch is that, the install with the patch will have its own toggle to disable/enable logging of history to Google's MyActivity.
This allows users to not have either YT or YTM history logged from a device / app without disabling it globally

**Reply by oSumAtrIX** [reply]
If YouTube tracks history server side, then the patch would have to disable the history and enable it when you exit the app, or open the other app. But it should be possible client side

**Reply by sohilsayed** [reply]
Actually if you disable one of the hosts of YouTube in things like adaway you disable the history
